lib/oli_web/components/delivery/user_account.ex

```diff
@@ -421,4 +406,16 @@ defmodule OliWeb.Components.Delivery.UserAccount do
         Routes.session_path(OliWeb.Endpoint, :signout, type: :user)
     end
   end
+
+  defp to_initials(%{name: nil}), do: "G"
+
+  defp to_initials(%{name: name}) do
+    name
+    |> String.split(" ")
+    |> Enum.take(2)
+    |> Enum.map(&String.slice(&1, 0..0))
+    |> Enum.join()
+  end
+
+  defp to_initials(_), do: "?"
```

Comment:
I couldn't see the criteria in the ticket for how to take the initials. Could you please put it here?